### PR TITLE
Add registration for Dirk Gomez

### PR DESCRIPTION
**Pull request by dirkgomez**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] I read the `THE IMPORTANT STUFF` at [https://jscraftcamp.org/registration](https://jscraftcamp.org/registration), the JSON file follows the [template](https://github.com/jscraftcamp/website/blob/main/participants/_template.json), and contains the mandatory fields like `realName`, `githubAccountName`, `when`, `iCanTakeNotesDuringSessions`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`.
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [x] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/5/views/1)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

File: participants/djg.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": false,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	"allergies": [],
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "I started with JavaScript when it was considered a toy language where you simply cut and paste code and last coded JavaScript with jQuery. These days I mostly use TypeScript for IAAC.",
```
Comment (Narigo) [inline participants/djg.json:31]: What is `IAAC`?

File: participants/djg.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "TypeScript in AWS CSK.",
```
Comment (Narigo) [inline participants/djg.json:33]: I only know AWS CDK - typo or is this another thing? 😅 I always get confused by abbreviations...

File: participants/djg.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		"familyName": "Gomez"
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "dirkgomez",
+	// either both days or at least one day has to be set to true
+	"when": {
+		// June 7th, 2024
+		"friday": true,
+		// June 8th, 2024
+		"saturday": false
```
Comment (Narigo) [inline participants/djg.json:16]: Is Friday only attendance on purpose?

**Review by Narigo — COMMENTED** [review]
Hi @dirkgomez , quick check before I merge: Is the Friday only attendance on purpose?

**Comment by dirkgomez** [comment]
> Hi @dirkgomez , quick check before I merge: Is the Friday only attendance on purpose?

No, I also correct the other mistakes. Thanks for your diligence.

**Comment by Narigo** [comment]
Welcome to the JSCC24 @dirkgomez ! Looking forward to meeting you there and most probably learn about AWS 😅

**Pull request merged**